Storage Bay/storageroom.cpp

```diff
@@ -0,0 +1,189 @@
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <cstdlib>
+#include <ctime>
+
+using namespace std;
+
+// Function to read text from a file and append a newline
+string readFromFile(const string& filename)
+{
+    ifstream file(filename);
+    if (!file)
+    {
+        cerr << "Error: Could not open file " << filename << endl;
+        return "";
+    }
+    string content;
+    getline(file, content, '\0'); // Read entire file content into string
+    file.close();
+    content += "\n"; // Append a newline to the content
+    return content;
+}
+
+// Player class
+class Player
+{
+public:
+    enum CopilotState { LOST, FOUND, INJURED };

```

Comment:
In the current classes.cpp Copilot has his own class with public variables injured and obtained. The enum for player is also a global variable, however I like the idea of switching it to a player variable. Not much would need to be adjusted and it is not currently important to any gameplay.